manifests/secrets.yaml

```diff
@@ -1,23 +1,27 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: wsc-app-sample-wproofreader-lic
+  name: wproofreader-app-lic
   labels:
+    helm.sh/chart: wproofreader-1.0.0
     app.kubernetes.io/name: wproofreader
-    app.kubernetes.io/instance: wsc-app-sample
-    app.kubernetes.io/version: "5.25.0"
+    app.kubernetes.io/instance: wproofreader-app
+    app.kubernetes.io/version: "5.34.1"
+    app.kubernetes.io/managed-by: Helm
 type: Opaque
 data:
-  license: "cVdlUnRZMTIz"
+  license: "FYVD0w46QhygxIPZ6Byo"
```

Comment:
I've checked this key and it doesn't look like a real key. Was it a fake one generated for the purpose of this Chart?

Reply:
Sure, it was fake key. 
Replaced with "your-license-key".